indexer/data_header.hpp

```diff
@@ -71,6 +72,7 @@ namespace feature
     /// @name Serialization
     //@{
     void Save(FileWriter & w) const;
+    void SaveRegionData(FileWriter & w) const;
```

Comment:
As for two methods, these are called in sequence but are independent: the second call may be skipped, and they don't have to be called sequentially. I can remove `SaveRegionData` requiring a caller to know they should use `GetData().Serialize()`, but I'd prefer to keep the convenience method.